storage/db-tool/src/backup_maintenance.rs

```diff
@@ -28,11 +28,18 @@ pub struct CompactionOpt {
     #[clap(long, default_value = "1")]
     pub transaction_file_compact_factor: usize,
     #[clap(flatten)]
-    pub metadata_cache_opt: MetadataCacheOpt,
+    pub metadata_cache_path: MetadataCacheOpt,
```

Comment:
why? `metadata_cache_opt.dir` is a bit awkward, true, but if we add a new field to MetadataCacheOpt, say `refresh`, `metadata_cache_path.refresh` will be look "wrong"